src/backend/utils/authUtils.js

```diff
@@ -4,10 +4,12 @@ import jwt_decode from "jwt-decode";
 
 export const requiresAuth = function (request) {
   const encodedToken = request.requestHeaders.authorization;
+  console.log(encodedToken)
   const decodedToken = jwt_decode(
     encodedToken,
     process.env.REACT_APP_JWT_SECRET
   );
+  console.log(decodedToken)
```

Comment:
If you have checked the token you can remove the console.log

Reply:
will do